### PR TITLE
RN Nav Added

### PR DESCRIPTION
**Pull request by Aashirbadd**
Added New Screens, Refactored Code, Added Navigation.

### CODE REVIEW

File: NoteAId-FE/app/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
```
Comment (mahdiva) [inline NoteAId-FE/app/components/Card.js:1]: I noticed some of these components are not being used in the app, so we should get rid of them. There are other ones such as `AuthNavigator`, `FeedNavigator`, `NewListingButton`, etc...

File: NoteAId-FE/app/screens/AccountScreen.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
```
Comment (mahdiva) [inline NoteAId-FE/app/screens/AccountScreen.js:1]: We can also get rid of the screens that are not being used in the main navigation including `ListingDetailsScreen`, `ListingsScreen`, `LoginScreen`, `RegisterScreen`, ...

**Review by mahdiva — CHANGES_REQUESTED** [review]
Everything is looking good!

There were just some extra components/navigations that are not being used in the app that we can get rid of, which I left some comments for. Some of the pictures under `app/assets` that are not used can also be deleted. But otherwise everything looks good.

Great job! :rocket:

**Comment by mahdiva** [comment]
Also I'm getting the following errors:
- Font Family "AvenirNext-Bold" is not a system font and has not been loaded through Font.loadAsync
- Font Family "AvenirNextCondensed-UltraLightItalic" is not a system font and has not been loaded through Font.loadAsync

And the following warning:
- `ImagePicker.requestCameraRollPermissionsAsync()` is deprecated in favour of `ImagePicker.requestMediaLibraryPermissionsAsync()`

File: NoteAId-FE/app/components/Card.js (hunk 1 above)
Comment (Aashirbadd) [reply NoteAId-FE/app/components/Card.js]: All the misc files have been removed now Mahdi!

File: NoteAId-FE/app/screens/AccountScreen.js (hunk 2 above)
Comment (Aashirbadd) [reply NoteAId-FE/app/screens/AccountScreen.js]: Removed now!

**Review by mahdiva — APPROVED** [review]

**Pull request merged**